### PR TITLE
Followup on #396

### PR DESCRIPTION
**Pull request by erickgirard**
Fixes the code style suggestion by @ktilcu at https://github.com/BlakeGuilloud/ganon/pull/406#issuecomment-338664757

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

For some reason it compares your master to the merge of this morning with my code. I'm not sure I understand and how to fix it.

Any suggestion ?

### CODE REVIEW

File: lib/takeWhile.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ * `takeWhile` take a slice of an array, elements will be taken 
+ * from the begining of the array until selector returns false
+ * 
+ * @params {Array} array from which values will be iterated
+ * @params {predicate} delegate function to determine when to stop iterating
```
Comment (ktilcu) [inline lib/takeWhile.js]: I'm sorry. I wasn't clear. Function is the right type. I was just suggesting delegate->predicate

**Comment by erickgirard** [comment]
I just pulled the changes from your repo and pushed them to my local copy, forgot to that the first time. Should now show correctly to only update the takeWhile doc/params :-)

**Comment by ktilcu** [comment]
@erickgirard Thanks so much and way to go! Could you create an issue for your new method? It would be super helpful.

**Comment by erickgirard** [comment]
Absolutely, will do right now. :-)

**Pull request merged**